server/lorax_server/models/__init__.py

```diff
@@ -234,7 +234,7 @@ def get_model(
                 trust_remote_code=trust_remote_code,
             )
 
-    if model_type == "llama":
+    if model_type in ["llama", "llama_aqlm"]:
```

Comment:
`llama_aqlm` and similar custom model types are deprecated. We moved AQLM from custom code to transformers integration with `transformers` `4.38.0`. Correct me if I'm wrong and it serves different purpose here.